wandb/sdk/launch/runner/gcp_vertex.py

```diff
@@ -207,13 +214,25 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:
         return submitted_run
 
 
-def run_shell(args: List[str]) -> str:
-    return subprocess.run(args, stdout=subprocess.PIPE).stdout.decode("utf-8").strip()
-
-
 def get_gcp_config(config: str = "default") -> Any:
     return yaml.safe_load(
         run_shell(
             ["gcloud", "config", "configurations", "describe", shlex_quote(config)]
-        )
+        )[0]
+    )
+
+
+def exists_on_gcp(image: str, tag: str) -> bool:
+    out, err = run_shell(
+        [
+            "gcloud",
+            "artifacts",
+            "docker",
+            "images",
+            "list",
+            shlex_quote(image),
```

Comment:
this image will have a `:` in it which will cause this image to be called as "'image:tag'", right? Does this command handle that okay?

Reply:
the image passed here does not contain `:`, not sure what you mean here

Reply:
Okay great, I thought that this was in the format "name:tag"